### PR TITLE
[TASK] Remove validations query in GET /validators

### PR DESCRIPTION
**Pull request by wilsonianb**

### CODE REVIEW

File: api/controllers/ValidatorsController.js [hunk 1]
```diff
@@ -12,13 +12,20 @@ module.exports = {
...
-        domain: verification.domain,
-        error: verification.error
+    // Return validators with their most recent domain status
+    let verifications = await database.Verifications.findAll({
+      order: [['"createdAt"', 'DESC']]
+    })
+    let validators = []
+    let validation_public_keys = []
+    for (let verification of verifications) {
+      if (validation_public_keys.indexOf(verification.validation_public_key)===-1) {
```
Comment (stevenzeiler) [inline api/controllers/ValidatorsController.js]: consider `_.has(validation_public_keys, verification.validation_public_key)` to be more readable.

**Comment by stevenzeiler** [comment]
:+1:

File: api/controllers/ValidatorsController.js (hunk 1 above)
Comment (wilsonianb) [inline api/controllers/ValidatorsController.js]: `_.has` appears to only be for objects. `validation_public_keys` is an array.

Comment (wilsonianb) [inline api/controllers/ValidatorsController.js]: Looks like es7 has `includes`, so I'll use that.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Comment (stevenzeiler) [inline api/controllers/ValidatorsController.js]: oh in that case I would use `_.contains(list, value)`

**Comment by wilsonianb** [comment]
switched to `_.contains`

**Pull request merged**